index.html

```diff
@@ -6,21 +6,30 @@
         <link rel="stylesheet" href="css/style.css">
 
         <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.8/angular.min.js"></script>
-        <script src="js/fruits_and_veggies_test.js"></script>
+        <script src="js/fruits_and_veggies.js"></script>
         <script src="js/app.js"></script>
     </head>
-    <body>
+    <body ng-app="FandV" ng-controller="appCtrl">
         <div class="container">
```

Comment:
Here is my HTML for the fruits:

    <div class="col-sm-4">
        Fruits
        <div ng-repeat="aFruit in allFruit track by $index" ng-class="{'has-fail': !win, 'has-fail': fail}" class="well">
            {{ aFruit }}
            <button ng-click="toPoolFromFruit($index)" type="button" class="btn btn-primary"><span class="glyphicon glyphicon-arrow-right"></span></button>
        </div>
    </div>